Makefile

```diff
@@ -124,6 +144,40 @@ openssl:
 		./Configure android-armv7 && \
 		ANDROID_DEV=$(SYSROOT)/usr make build_libs
 
+.PHONY: opus
+opus:
+	cd opus && \
+		rm -rf include_opus && \
+		CC="arm-linux-androideabi-gcc --sysroot $(SYSROOT)" \
```

Comment:
using arm-linux-androideabi-gcc assumes that the compiler is in PATH.

can we use the full path instead:

```
CC="$(CC) --sysroot $(SYSROOT)"
```